OracleInstantClient/README.md

```diff
@@ -1,6 +1,6 @@
-# About this Docker Image
+# About this Container Image
```

Comment:
Neither container nor image are proper nouns so neither of them should be capitalised.

Reply:
You're supposed to be away aren't you? It's a heading IMHO.  Hence the '#'

Reply:
Oracle standard is sentence case for headings which means only the first word is capitalised.

Reply:
See Slack